include/bluetooth/direction.h

```diff
@@ -142,6 +142,19 @@ struct bt_df_conn_iq_samples_report {
 	/** Pinter to IQ samples data. */
 	struct bt_hci_le_iq_sample const *sample;
 };
+
+struct bt_df_conn_cte_req_params {
+	/** Requested interval for initiating the CTE Request procedure. Value 0x0 means, run the
+	 * procedure once. Other values are intervals in number of connection events, to run the
+	 * command periodically.
+	 */
```

Comment:
That's quite a long brief. Consider
```suggestion
	/** 
	* @brief Requested interval for initiating the CTE Request procedure. 
	* 
	* Value 0x0 means, run the
	 * procedure once. Other values are intervals in number of connection events, to run the
	 * command periodically.
	 */
```

Is `interval` the name used by the spec? Otherwise I think it is misguiding since interval is usually expressed in a time unit, but this is number of connection events.

Reply:
`interval` refers to `CTE_Request_Interval` from the BT 5.3 Core spec.